XRTK-Core/Packages/com.xrtk.core/Definitions/Controllers/Simulation/SimulatedControllerDataProviderProfile.cs

```diff
@@ -118,5 +118,10 @@ public class SimulatedControllerDataProviderProfile : BaseMixedRealityController
         public float RotationSpeed => rotationSpeed;
 
         #endregion
+
+        public override ControllerDefinition[] GetDefaultControllerOptions()
+        {
+            return null;
```

Comment:
need to figure out what this is

Reply:
this is correct. In the simulated hands profile provider we correctly create the `SimulatedHandController`s for both left and right.